app/uk/gov/hmrc/exports/repositories/DeclarationRepository.scala

```diff
@@ -33,6 +33,7 @@ class DeclarationRepository @Inject()(mc: ReactiveMongoComponent, appConfig: App
       ExportsDeclaration.Mongo.format,
       objectIdFormats
     ) {
+  def update(submittedDeclaration: Any) = ???
```

Comment:
I've implemented this now so you can revert this class

Reply:
I've merged in master for you @PauloPortugal

Reply:
Looks like it's been overwritten...

Reply:
`rebase` vs `merge` sorry my fault there.